Source/Public/Views/TvOSMoreButton.swift

```diff
@@ -40,6 +51,17 @@ open class TvOSMoreButton: UIView {
         }
     }
 
+    var focusableMode: FocusableMode = .auto {
+        didSet {
+            switch focusableMode {
+            case .manual(let isFocusable):
+                self.isFocusable = isFocusable
+            default:
+                isFocusable = false
```

Comment:
The following example fails because `isFocusable` is set to false when using `auto`:

```swift
// Set Long text
tvOSMoreButton.focusableMode = .manual(false)
tvOSMoreButton.focusableMode = .auto
// Button is not focused even though focusable mode is auto.
```

```diff
-            default:
-                isFocusable = false
+            case .auto:
+                updateUI()
```